jubjub/src/fr.rs

```diff
@@ -70,6 +78,33 @@ pub const MODULUS: Fr = Fr([
     0x0e7d_b4ea_6533_afa9,
 ]);
 
+const MODULUS_BYTES: [u8; 32] = [
```

Comment:
Say what byte order this is in (maybe include it in the name).

Reply:
Also consider renaming `Fr::from_bytes` to `from_le_bytes`.

Reply:
See the other comment; the same holds here.